game/views.py

```
@@ -49,8 +60,11 @@ def index(request):
 
 def record_game_result(request, mode, winner, reason, player_color='white'):
     """Save a completed game result to the database."""
-    user = request.user if request.user.is_authenticated else None
-    GameResult.objects.create(user=user, mode=mode, winner=winner, end_reason=reason, player_color=player_color)
+    try:
+        user = request.user if request.user.is_authenticated else None
+        GameResult.objects.create(user=user, mode=mode, winner=winner, end_reason=reason, player_color=player_color)
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)